qt/scientific_interfaces/Indirect/IndirectDataAnalysisTab.cpp

```diff
@@ -22,7 +24,7 @@ namespace IDA {
 IndirectDataAnalysisTab::IndirectDataAnalysisTab(QWidget *parent)
     : IndirectTab(parent), m_dblEdFac(nullptr), m_blnEdFac(nullptr),
       m_parent(nullptr), m_inputWorkspace(), m_previewPlotWorkspace(),
-      m_selectedSpectrum(0) {
+      m_selectedSpectra(0) {
```

Comment:
Does this want to be `Spectra` or `Spectrum`? It seems to me as though `m_selectedSpectrum` refers to a single spectrum, so calling it `Spectra` is confusing. Or am I missing something?

Reply:
Some knowledge of the science is needed for these Guesses. Also, the data used is not appropriate for some of them. The purpose of plotting the Guess is to find suitable values by trial & error!
Moving the bars for the width in should sharpen up the peaks, though I have had problems previously with 2 Lorentzians.